ProcessesApi.Tests/V1/E2ETests/Steps/UpdateChangeOfNameProcessSteps.cs

```diff
@@ -0,0 +1,131 @@
+using FluentAssertions;
+using Hackney.Core.Sns;
+using Hackney.Core.Testing.DynamoDb;
+using Hackney.Core.Testing.Shared.E2E;
+using Hackney.Core.Testing.Sns;
+using Newtonsoft.Json;
+using ProcessesApi.Tests.V1.E2ETests.Steps.Constants;
+using ProcessesApi.V1.Boundary.Constants;
+using ProcessesApi.V1.Boundary.Request;
+using ProcessesApi.V1.Infrastructure;
+using ProcessesApi.V1.Infrastructure.JWT;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace ProcessesApi.Tests.V1.E2ETests.Steps
+{
+    public class UpdateChangeOfNameProcessSteps : BaseSteps
+    {
+        private readonly IDynamoDbFixture _dbFixture;
```

Comment:
Is this similar to UpdateSTJProcessSteps? Maybe make a UpdateProcess base class with shared methods?